fineract-provider/src/main/java/org/apache/fineract/portfolio/loanproduct/domain/LoanProductTrancheDetails.java

```diff
@@ -55,6 +59,15 @@ public void update(final JsonCommand command, final Map<String, Object> actualCh
             this.multiDisburseLoan = newValue;
         }
 
+        if (command.isChangeInBooleanParameterNamed(LoanProductConstants.REVOLVING_LOAN_PARAMETER_NAME, this.isRevolving)) {
+            final boolean newValue = command.booleanPrimitiveValueOfParameterNamed(LoanProductConstants.MULTI_DISBURSE_LOAN_PARAMETER_NAME);
+            actualChanges.put(LoanProductConstants.REVOLVING_LOAN_PARAMETER_NAME, newValue);
+            this.isRevolving = newValue;
+        }
+
+        // XXX TODO: maybe check that there is no way for creating a product that is revolving but don't have
```

Comment:
if we don't set multiDisbursement then the effect of the revolving period will be a range when we can do disbursement (only once). So maybe it is ok not having this kind of check

Reply:
seems like an unintended effect though...